modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/MultiSelectTag.java

```diff
@@ -214,6 +218,29 @@ protected int processStartTag() throws Exception {
 
 		jspWriter.write(" type=\"text\" /></div>");
 
+		if (!_disabled && !_disabledClearAll && !selectedItems.isEmpty()) {
+			jspWriter.write("<div class=\"input-group-item ");
+			jspWriter.write("input-group-item-shrink\">");
+
+			ButtonTag buttonTag = new ButtonTag();
+
+			buttonTag.setCssClass("component-action");
+			buttonTag.setDisplayType("secondary");
+			buttonTag.setIcon("times-circle");
+
+			if (Validator.isNotNull(_clearAllTitle)) {
+				String clearAllTitle = LanguageUtil.get(
+					TagResourceBundleUtil.getResourceBundle(pageContext),
+					_clearAllTitle);
+
+				buttonTag.setTitle(HtmlUtil.escape(clearAllTitle));
```

Comment:
@kresimir-coko after a second thought, I'm not 100% sure we should escape the title here.
I'm leaving a comment so we don't forget talking about this with @carloslancha and @markocikos.

Reply:
Hey @julien, why aren't you 100% sure if we should or should not escape here? 😉 

Could you please add your reasoning here so @kresimir-coko understands why maybe not?

Reply:
Yes, @jbalsas that's a good point.

My reasoning was the following: 

I don't think we're going to allow users to set a value from the UI for this `clearAllTitle` property, and if someone using the tag from a jsp does something like this:

```jsp
<clay:multi-select
     clearAllTitle="<script>alert(123)</script>"
/>
```

 I guess they're looking for trouble; but that's what I'm 100% sure about .

Reply:
Shouldn't it be job of the `ButtonTag` to escape? :thinking: If I understood recent discussion correctly, we wish to escape as close as possible to the endpoint.

Reply:
> Shouldn't it be job of the ButtonTag to escape? 🤔 If I understood recent discussion correctly, we wish to escape as close as possible to the endpoint.

Yup!

Reply:
We fixed this in [`9dd5f5e`](https://github.com/liferay-frontend/liferay-portal/pull/528/commits/9dd5f5eb41dfc6410befe20c3b16e93e3ff35f9e)